app/src/main/java/com/sun/gamevui/ui/NewsDetailFragment.kt

```diff
@@ -1,21 +1,36 @@
 package com.sun.gamevui.ui
 
+import android.webkit.WebViewClient
+import androidx.navigation.fragment.findNavController
+import androidx.navigation.fragment.navArgs
 import com.sun.gamevui.R
 import com.sun.gamevui.base.BaseFragment
 import com.sun.gamevui.databinding.FragmentNewsDetailBinding
 import org.koin.androidx.viewmodel.ext.android.viewModel
 
 class NewsDetailFragment : BaseFragment<FragmentNewsDetailBinding>() {
 
+    private val arg: NewsDetailFragmentArgs by navArgs()
+
     override val layoutId get() = R.layout.fragment_news_detail
     override val viewModel by viewModel<NewsViewModel>()
 
     override fun initViews() {
     }
 
     override fun initData() {
+        binding?.apply {
+            lifecycleOwner = viewLifecycleOwner
+            webNews.apply {
+                webViewClient = WebViewClient()
+                loadUrl(arg.link)
+            }
```

Comment:
không cache vào à em

Reply:
web view mà muốn pull to refresh thì làm như nào ?

Reply:
lồng bên ngoài swipeRefreshlayout hả a
xong handle refresh listenner